core/internal/runbranch/resume_test.go

```diff
@@ -1305,9 +1305,8 @@ func TestExtractRunStateAdjustsStartTime(t *testing.T) {
 		"must")
 
 	// Set a non-zero StartTime in the input RunParams
-	initialStartTime := time.Now()
 	params := &runbranch.RunParams{
-		StartTime: initialStartTime,
+		StartTime: time.Now(),
```

Comment:
Does this need to be initialized at all for the test?

Reply:
I think we probably can remove it